### PR TITLE
[PPML] Add TPC-H example

### PR DESCRIPTION
**Pull request by jenniew**
Add TPC-H example to BigDL PPML

### CODE REVIEW

**Comment by jason-dai** [comment]
Need to add reference and credit for where the code is adapted from

File: scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/examples/tpch/Q01.scala [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/*
```
Comment (jason-dai) [inline scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/examples/tpch/Q01.scala:1]: Did we make any changes to the files?

Comment (jenniew) [reply scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/examples/tpch/Q01.scala:1]: no change for Q01-Q22.
modified TPCHQuery and TPCHSchemaProvider

Comment (jason-dai) [reply scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/examples/tpch/Q01.scala:1]: > no change for Q01-Q22. modified TPCHQuery and TPCHSchemaProvider

If there no change, you need to add "This file is copied from ..." at the beginning

If the file is changed, you may refer to an example at https://github.com/intel-analytics/BigDL/blob/main/python/orca/example/learn/tf2/yolov3/__init__.py

Comment (jenniew) [reply scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/examples/tpch/Q01.scala:1]: Our scala style check only allow BigDL license in the header.  So maybe we split the original license to second part? Or we change our scala style check not contain ending "*/" in the header?

Comment (jason-dai) [reply scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/examples/tpch/Q01.scala:1]: just exclude these files in style check

Comment (jenniew) [reply scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/examples/tpch/Q01.scala:2]: ok

**Comment by jenniew** [comment]
jenkins pass: http://10.112.231.51:18889/job/BigDL-PR-scala-ppml-unittests-Validation/201/

**Pull request merged**